rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -716,13 +716,13 @@ def _get_docs_for_batch(
 
         return batch_docs
 
-    def train(
+    def process_training_data(
         self,
         training_data: TrainingData,
         config: Optional[RasaNLUModelConfig] = None,
         **kwargs: Any,
     ) -> None:
```

Comment:
@wochinge what should this function return?

Reply:
just the training data that you just modified in that function :)  -- in the graph definition, each of these functions becomes kind of a node with the parameters being the input and the output of the function begin passed on to another node (i.e. we cannot tell the graph that something is just modified in place, we must return a value and it can then pass on this value to whatever other node is definied in the graph definition)